packages/rum-core/src/performance-monitoring/transaction-service.js

```diff
@@ -66,10 +66,19 @@ class TransactionService {
       const tr = this.getCurrentTransaction()
       if (tr && tr.captureTimings) {
         let capturePaint = false
+        /**
+         * For page load transactions, we capture all web vitals
+         * metrics including paint timings data and also the
+         * relative start time of transaction is set to zero
+         * to include all the buffered events
+         */
         if (tr.type === PAGE_LOAD) {
           capturePaint = true
         }
-        const { spans, marks } = captureObserverEntries(list, { capturePaint })
+        const { spans, marks } = captureObserverEntries(list, {
+          capturePaint,
+          trStart: capturePaint ? 0 : tr._start
```

Comment:
I think we should use `tr.type === PAGE_LOAD` , since capturePaint is not relevant here.

Reply:
I am opening a new PR to refactor this bit, we can use a different flag as capturePaint was added long before when we had only paint metrics.